src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,127 @@
+import { Link, useParams } from 'react-router-dom';
+import { usePeople } from '../store/PeopleContext';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { useFilters } from '../store/FilterContext';
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
+
+function isPersonFemale(person: Person | undefined): boolean {
+  return person?.sex === 'f';
+}
```

Comment:
The `isPersonFemale` function checks if a person is female. It currently handles `undefined` by returning `false`, which is fine, but consider adding a comment to clarify this behavior for future maintainability.